docs/more-pushmsg.md

```diff
@@ -154,28 +148,34 @@ public class GotifyClient {
     }
 
     private final String gotifyUrl;
+    private final HttpClient httpClient;
+    private final ObjectMapper objectMapper;
 
-    public GotifyClient(String gotifyUrl) {
-        this.gotifyUrl = gotifyUrl;
+    public GotifyClient(String baseUrl, String token) {
+        this.gotifyUrl = String.format("%s/message?token=%s", baseUrl, token);
+        this.httpClient = HttpClient.newHttpClient();
```

Comment:
Nice, didn't know java introduced a new http client in java 11.

Reply:
Yes, it has a cleaner API, HTTP/2 support and a few other features.

There's a lot pretty cool things happening to Java recently. I can only recommend taking a look at all the [JEPs](http://openjdk.java.net/projects/jdk) from version 11 to 16.